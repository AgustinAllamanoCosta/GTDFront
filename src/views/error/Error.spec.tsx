import React from 'react';
import { expect } from '@jest/globals';
import { render } from '@testing-library/react';
import ErrorView from './Error';
import { BrowserRouter } from 'react-router-dom';

describe('Error View', () => {
  test('Should render an Error view ', async () => {
    const { container } = render(
      <BrowserRouter>
        <ErrorView
          onClick={() => {}}
          message="some error message"
        />
      </BrowserRouter>,
    );
    expect(container).toMatchSnapshot();
  });
});
