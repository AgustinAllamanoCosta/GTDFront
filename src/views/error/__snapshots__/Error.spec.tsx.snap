// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error View Should render an Error view  1`] = `
<div>
  <div
    class="sc-dsLQwm ehlGIL"
  >
    <div
      class="sc-iKTcqh iCNJAu"
    >
      <div
        class="sc-braxZu bzYKie"
        data-cy="Card-Contents"
        is_center="true"
        padding="true"
      >
        <span
          class="sc-la-DxNn kUqJKy"
          data-cy="Error-view-title"
        >
          Something went wrong!
        </span>
        <span
          class="sc-gnpbhQ ksXoUd"
          data-cy="Error-view-message"
        />
        <div
          class="sc-gJhJTp QnPUY"
          data-cy="button-OK"
          data-testid="button-OK"
        >
          <span
            class="sc-elDIKY eIxNWV"
            data-cy="button-text"
          >
            OK
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
